amenity-points.mss

```diff
@@ -679,16 +679,17 @@
   }
 
   [feature = 'amenity_recycling'][recycling_type = 'centre'][zoom >= 17],
-  [feature = 'amenity_recycling'][zoom >= 19] {
+  [feature = 'amenity_recycling'][recycling_type != 'centre'][zoom >= 19] {
     marker-file: url('symbols/amenity/recycling.svg');
-    marker-fill: @amenity-brown;
+  [recycling_type = 'centre'] { marker-fill: @amenity-brown; }
+  [recycling_type != 'centre'] { marker-fill: @man-made-icon; }
     marker-placement: interior;
     marker-clip: false;
-    [access != ''][access != 'permissive'][access != 'yes'] {
-      marker-opacity: 0.33;
-    }
+  [access != ''][access != 'permissive'][access != 'yes'] {
+    marker-opacity: 0.33;
   }
```

Comment:
It looks like you accidentally loose this indentation, access section should start on the same level as the marker-clip.